library/src/main/java/com/bumptech/glide/request/target/NotificationTarget.java

```diff
@@ -61,6 +67,9 @@ public NotificationTarget(
    * @param notificationTag The notificationTag of the Notification that we want to load the Bitmap.
    *     May be {@code null}.
    */
+  @SuppressLint("InlinedApi")
```

Comment:
Why do you need to suppress inlined API?

Reply:
See PR OP.